### ISSUE TITLE
Extend Mustache

### ISSUE THREAD

**Issue by czukowski** [zombor/KOstache#13]
What do you think about creating an empty class, for example `class Kohana_Mustache extends Mustache` and call `Kohana_Mustache` from KOstache to allow people to customize it.

To explain, back when KOstache extended Mustache directly, I've had this method overridden:

```
protected function _renderEscaped($tag_name)
{
    return HTML::chars($this->_getVariable($tag_name), FALSE);
}
```

Mustache's own method was using `htmlentities()` which has done unnecessary work, since Kohana is UTF-8 only.

This would also be a convenient place to add any other custom code to Mustache.

**Comment by zombor** [comment]
+1

**Comment by shadowhand** [comment]
Fixed, see #14.
